hack/scale_out_poc/setup_nginx_proxy.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+# Copyright 2020 Authors of Arktos.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+print_help() {
+        echo "This is a tool to set up nginx proxy for Arktos scale-out tests in the local host."
+        echo 
+        echo "To run it: "
+        echo "    TENANT_PARTITION_IP=##.##.##.## RESOURCE_PARTITION_IP=##.##.##.## setup_nginx_proxy.sh"
```

Comment:
for multiple partitions, this can be defined as array-like string, e.g. comma separated strings.

Reply:
This script does not have the capability to handle the multiple clusters in either partitions. Once we figure out the way to support multi-partitions, this script will be updated.